src/pages/listInformation/components/ListInformationMenu/ListInformationMenu.tsx

```diff
@@ -95,6 +96,17 @@ export const ListInformationMenu: React.FC<ListInformationMenuProps> = ({
     );
   }
 
+  if (stripes.hasPerm(USER_PERMS.UpdateList)) {
+    actionButtons.push(
+      {
+        label: 'copy',
+        icon: ICONS.duplicate,
+        onClick: buttonHandlers.copy,
+        disabled: isEditDisabled(conditions),
```

Comment:
Should the duplicate button be disabled if editing is disabled? It seems like at least some of the reason that we'd disabling editing wouldn't really apply to duplicating